style.css

```diff
@@ -1,85 +1,112 @@
-    justify-content: center;
-    min-height: 100vh;
-    font-family: "Indie Flower", cursive;
-    font-weight: bold;
-    color: #272727;
+  background: #fb923c;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  min-height: 100vh;
+  font-family: "Indie Flower", cursive;
+  font-weight: bold;
+  color: #272727;
 }
 
 .container {
-    width: 80vw;
-    max-width: 500px;
+  width: 80vw;
+  max-width: 500px;
 }
 
 .heading {
-    background: var(--page-color);
-    text-align: center;
-    box-shadow: 4px 4px 4px #403f3e;
-    border-top-left-radius: 10px;
-    border-top-right-radius: 10px;
-    display: grid;
-    grid-template-columns: repeat(12, 1fr);
-    border-bottom: 5px double var(--line-color);
+  background: var(--page-color);
+  text-align: center;
+  box-shadow: 4px 4px 4px #403f3e;
+  border-top-left-radius: 10px;
+  border-top-right-radius: 10px;
+  display: grid;
+  grid-template-columns: repeat(12, 1fr);
+  border-bottom: 5px double var(--line-color);
 }
 
 .top-left-empty {
-    grid-column: 1 / 3;
+  grid-column: 1 / 3;
 }
 
 .top-left-empty,
 .margin {
-    border-right: 5px double var(--line-color);
+  border-right: 5px double var(--line-color);
 }
 
 .heading-text {
-    margin: 0.4em 0;
-    grid-column: 3 / 13;
-    display: flex;
-    align-items: flex-end;
-    justify-content: center;
-    font-size: 30px;
-    letter-spacing: -1px;
+  margin: 0.4em 0;
+  grid-column: 3 / 13;
+  display: flex;
+  align-items: flex-end;
+  justify-content: center;
+  font-size: 30px;
+  letter-spacing: -1px;
 }
 
 .page {
-    border-top-left-radius: 0;
-    border-top-right-radius: 0;
-    border-bottom-left-radius: 10px;
-    border-bottom-right-radius: 10px;
-    box-shadow: 4px 4px 4px #403f3e;
-    min-height: calc(var(--line-height) * 10);
-    font-size: 20px;
-    background: repeating-linear-gradient(
-        var(--page-color) 0 calc(var(--line-height) - 2px),
-        var(--line-color) calc(var(--line-height) - 2px) var(--line-height)
-    );
-    line-height: var(--line-height);
-    display: grid;
-    grid-template-columns: repeat(12, 1fr);
+  border-top-left-radius: 0;
+  border-top-right-radius: 0;
+  border-bottom-left-radius: 10px;
+  border-bottom-right-radius: 10px;
+  box-shadow: 4px 4px 4px #403f3e;
+  min-height: calc(var(--line-height) * 10);
+  font-size: 20px;
+  background: repeating-linear-gradient(
+    var(--page-color) 0 calc(var(--line-height) - 2px),
+    var(--line-color) calc(var(--line-height) - 2px) var(--line-height)
+  );
+  line-height: var(--line-height);
+  display: grid;
+  grid-template-columns: repeat(12, 1fr);
 }
 
 .margin {
-    grid-column: 1 / 3;
-    text-align: center;
+  grid-column: 1 / 3;
+  text-align: center;
 }
 
 .text-area {
-    grid-column: 3 / 13;
-    padding: 0 10px;
+  grid-column: 3 / 13;
+  padding: 0 10px;
+}
+
+#btn {
+  background: #ff4742;
+  border: 1px solid #ff4742;
```

Comment:
set `border: none`, the red border doesn't look good when the background is dark during hover.